packages/components/stories/icon/icons.js

```diff
@@ -13,7 +13,7 @@ export const icons =
         'IcBrandDmt5',
```

Comment:
TODO: revert changes to this file

Reply:
File contains new entries - IcCurrencyEurCheck & IcCurrencyEurCheck